src/test/java/io/asyncer/r2dbc/mysql/IntegrationTestSupport.java

```diff
@@ -38,10 +39,33 @@ abstract class IntegrationTestSupport {
 
     private final MySqlConnectionFactory connectionFactory;
 
+    private static final MySQLContainer<?> container;
+
     IntegrationTestSupport(MySqlConnectionConfiguration configuration) {
         this.connectionFactory = MySqlConnectionFactory.from(configuration);
     }
 
+    static {
+        String password = System.getProperty("test.mysql.password");
+        String version = System.getProperty("test.mysql.version");
```

Comment:
How about to use a fallback version(8), if the version is not explicitly specified?